encfs/RawFileIO.cpp

```diff
@@ -220,14 +221,17 @@ bool RawFileIO::write(const IORequest &req) {
   ssize_t bytes = req.dataLen;
   off_t offset = req.offset;
 
+  int eno = 0;
   while ((bytes != 0) && retrys > 0) {
+    errno = 0;
```

Comment:
No as we need `eno` on line 247 : if we exit loop because we did not write all data (`retrys == 0)`, it is nice to check `errno`, we never know.